maven/detect_test.go

```diff
@@ -48,9 +48,32 @@ func testDetect(t *testing.T, context spec.G, it spec.S) {
 		Expect(os.RemoveAll(ctx.Application.Path)).To(Succeed())
 	})
 
-	it("fails without pom.xml", func() {
+	it("only provides with pom.xml", func() {
```

Comment:
```suggestion
	it("only provides without pom.xml", func() {
```
Should this read without?

Reply:
I pushed a change to address this. There were some other minor issues with the detect test. Good catch. Thanks